UQ-eLIPSE/Crucible-Quiz
# Feature/vethub 326

convert base64 to image file
to test: 'yarn test'
to verify:
1. `yarn dev` run up the project
2. click "Create D&D quiz" button, choose "Text D&D question", then hand input text and click "convert" to image, at the console.log you will see the msg: 
![image](https://github.com/user-attachments/assets/8a146143-fa6a-429f-a19f-041265516d46)
